### ISSUE TITLE
[BUG] Refreshing the web/device view page fails to load any content

### ISSUE THREAD

**Issue by Parlane** [AnalogJ/scrutiny#264]
**Describe the bug**
If you click to navigate to a specific device and then press f5 on the device page a constant scrutiny logo appears and nothing loads

I am using the current ghcr.io/analogj/scrutiny:beta-omnibus image: 

**Expected behavior**
Refresh works normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/86614/170602257-b7854e6a-fab7-4ea5-bad0-170317259019.png)

**Log Files**

Chrome console:

```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec. polyfills-es2015.5483988857642b9a2626.js:1
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec. main-es2015.936a570b2df09daac3f4.js:1
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

**Comment by Parlane** [comment]
Sorry if this is a beta only bug. Thought I'd point it out anyway.

**Comment by Parlane** [comment]
Same problem happens if you middle click the device on the main page to open it in a new tab.

**Comment by AnalogJ** [comment]
Great catch, this looks like its related to the reverse proxy related fixes. #136 #208 
Its present in the master branch as well 😞 
I'll take a look.

**Comment by AnalogJ** [comment]
Hey @Parlane can you pull the latest beta image and verify that it works for you?

**Comment by AnalogJ** [comment]
fixed in v0.4.8 🎉

**Comment by Parlane** [comment]
I can confirm it is fixed in the latest beta-omnibus docker image :) 

Thanks @AnalogJ  🎉
